osbs/build/plugins_configuration.py

```diff
@@ -219,6 +219,29 @@ def adjust_for_custom_base_image(self):
         for when, which in plugins:
             self.pt.remove_plugin(when, which, msg)
 
+    def adjust_for_flatpak(self):
+        """
+        Remove plugins that don't work when building Flatpaks
+        """
+        if self.user_params.flatpak.value:
+            remove_plugins = [
+                ("prebuild_plugins", "resolve_composes"),
```

Comment:
In other plugins I've seen `atomic_reactor.constants` used instead of hardcoded names. Would be better to keep constants IMO

Reply:
that would mean dependency on atomic_reactor

Reply:
Ahh, right, I forgot I'm in client